requirements/default.txt

```diff
@@ -7,6 +7,9 @@ configman==1.3.0 \
     --hash=sha256:bc96b4c974aefabf4605d82dfab409a5aed10b4ae0b05ae2535eda415a3f7b67 \
     --hash=sha256:8e45c20c7b73173bd4d0194521f982201b70558d6f6e945f11053201110b83e0 \
     --hash=sha256:4000037487d65ec05e3d4183298f0a5799d7c71a47e8753f4edeede586baeb50
+click==7.0 \
+    --hash=sha256:2335065e6395b9e67ca716de5f7526736bfa6ceead690adf616d925bdc622b13 \
+    --hash=sha256:5b94b49521f6456670fdb30cd82a4eca9412788a93fa6dd6df72c94d5a8ff2d7
```

Comment:
`sqs_cli` uses click, so we need this to be a default dependency. I think I want to rewrite the other scripts to also use click, but that's a future low-priority task.